main.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 // Version holds the current Gitea version
-var Version = "1.0.0+dev"
+var Version = "1.0.2+dev"
```

Comment:
I forgot what the policy was, maybe it could be written in a comment above the number.
Is `1.0.2+dev` semantically greater than `1.0.2` ?